api/src/client/AirtableFeedbackClient.ts

```diff
@@ -0,0 +1,52 @@
+import { LogWriterType } from "@libs/logWriter";
+import { FeedbackRequest } from "@shared/feedbackRequest";
+import { UserData } from "@shared/userData";
+import Airtable from "airtable";
+import { FeedbackClient } from "../domain/types";
+
+type AirtableConfig = {
+  apiKey: string;
+  baseId: string;
+  baseUrl: string;
+};
+
+export const AirtableFeedbackClient = (config: AirtableConfig, logWriter: LogWriterType): FeedbackClient => {
+  const table = "tblHZp9vjRIdooR4Z";
+
+  Airtable.configure({
+    endpointUrl: config.baseUrl,
+    apiKey: config.apiKey,
+  });
+
+  const base = Airtable.base(config.baseId);
+
+  const create = (feedbackRequest: FeedbackRequest, userData: UserData): Promise<boolean> => {
+    return new Promise((resolve, reject) => {
+      const fields = {
+        fldEnXB3SBrgib5sD: feedbackRequest.detail,
+        fldcCp3dxMI5Ntn2m: userData.user.email,
+        fld5nU9Jeldk7bW8Y: userData.profileData.industryId,
+        fldEPNY8Aub2I8WEf: userData.profileData.hasExistingBusiness ? "owning" : "starting",
+        fldvKgl7DYMSXZTsv: feedbackRequest.pageOfRequest,
+        fldMGEB5JoyVIZhuD: feedbackRequest.device,
+        fldN1mp5PrXQLt1Fg: feedbackRequest.browser,
+        fldlo19Z2Ud1p0R24: feedbackRequest.screenWidth,
```

Comment:
These should all be real field names, similar to the User Testing airtable

Reply:
Faraz and I talked about this when we implemented it, and discovered that we can reference fields by IDs instead of name (both here and in the table name). This seems more robust because now it will work, even if the field names get tweaked without us realizing.

We actually created a chore to switch the User Testing airtable to use the IDs instead of names also: https://github.com/newjersey/navigator.business.nj.gov/issues/2825

Reply:
I was thinking we need 2 tables, 1 for dev/staging and 1 for prod. That way we can test without polluting the real data
Hence why we would want to do this by name. Same for the UserTesting table.

Reply:
Hm 🤔  that's a good point

Reply:
I'm gonna switch the chore I linked to be a chore to create separate dev/prod tables and start using them. For this code, I think Dave is right, we should be using the table name and the field names so that we can switch seamlessly between tables